templates/zerver/emails/find_team.html

```diff
@@ -22,6 +23,19 @@
 <p>
     {% trans %}If you have trouble logging in, you can <a href="{{ help_reset_password_link }}">reset your password</a>.{% endtrans %}
 </p>
+{% else %}
+<p>
+    {{ _("You have requested a list of Zulip accounts for this email address.") }}
+    {% if corporate_enabled %}
+    {{ _("Unfortunately, no accounts in Zulip Cloud organizations were found.") }}
```

Comment:
`Unfortunately, no Zulip Cloud accounts were found.` would be a bit cleaner here.